vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/order_handler.rb

```diff
@@ -0,0 +1,30 @@
+module SecureRooms
+
+  module AccessHandlers
+
+    class OrderHandler
+
+      def self.process(occupancy)
+        return unless occupancy.orderable?
+
+        order = Order.create!(
```

Comment:
As we add the extra method calls, it might be worth moving this over into instance methods.